assets/sawtooth_logo_lightblue.svg

```diff
@@ -0,0 +1 @@
2000/svg" viewBox="0 0 638.22 108.55"><defs><style>.cls-1{fill:#1272b9;}.cls-2{fill:#5dc5f0;}</style></defs><title>SAWTOOTH LOGO_LIGHTBLUE(1)</title><path class="cls-1" d="M578.7,161.1,555,178.44v-25l-34.28,25v-25l-23,16.82a40.86,40.86,0,0,0,81.14-6.89C578.76,162.59,578.74,161.84,578.7,161.1Z" transform="translate(-76.98 -95.65)"/><path class="cls-2" d="M578.08,155.88a40.86,40.86,0,0,0-81,7.46c0,.54,0,1.09,0,1.63l28.17-20.57v25l34.28-25v25Z" transform="translate(-76.98 -95.65)"/><path class="cls-2" d="M141.39,177.27q0,11.4-8.39,18.29t-21.3,6.89a56.88,56.88,0,0,1-17.93-3.09A52.69,52.69,0,0,1,77,190.25l9.47-13.48q12,9.31,25.68,9.32a11.64,11.64,0,0,0,6.6-1.65,5.07,5.07,0,0,0,2.44-4.37c0-1.82-1.13-3.49-3.37-5a38.25,38.25,0,0,0-9.76-4.31q-6.39-2-9.76-3.44a56.5,56.5,0,0,1-7.67-4.16q-8.76-5.46-8.75-16.14a21.13,21.13,0,0,1,8.61-17.43q8.59-6.75,22.23-6.75T140,132l-8,14.34q-10.05-6.89-20-6.88t-10,5.45c0,2,1,3.58,3.16,4.73a63.28,63.28,0,0,0,10.47,4q7.32,2.29,10.4,3.51a52.18,52.18,0,0,1,7.39,3.81Q141.4,165.87,141.39,177.27Z" transform="translate(-76.98 -95.65)"/><path class="cls-2" d="M218.68,201.3H198.46V192q-8.33,10.49-20.59,10.48t-20.59-7.11Q149,188.25,149,176.41t8.61-17.72q8.61-5.88,23.53-5.88h16.06v-.43q0-12.34-13-12.34a35.36,35.36,0,0,0-11.84,2.22,38.59,38.59,0,0,0-10.54,5.53l-9.61-13.92a57.58,57.58,0,0,1,34.72-11.05q14.06,0,22.95,7t8.89,22.24ZM197,171.89v-3.73H183.54q-12.91,0-12.92,8a7.4,7.4,0,0,0,3.09,6.39q3.07,2.22,8.82,2.22a15.67,15.67,0,0,0,10.12-3.52A11.44,11.44,0,0,0,197,171.89Z" transform="translate(-76.98 -95.65)"/><path class="cls-2" d="M271.59,201.3H249.35L223.1,124.12h22.24l15.06,46.62,15.06-46.62h20.8l15.07,46.62,15.06-46.62h22.38L322.52,201.3H300.14l-14.2-42Z" transform="translate(-76.98 -95.65)"/><path class="cls-2" d="M489.93,162.71A38.19,38.19,0,0,1,478.31,191q-11.62,11.47-29.2,11.48T419.92,191Q408.3,179.5,408.3,162.71a38.41,38.41,0,0,1,11.62-28.34q11.61-11.53,29.19-11.55t29.2,11.55A38.44,38.44,0,0,1,489.93,162.71Zm-60,0q0,9.47,5.38,15.42a18.91,18.91,0,0,0,27.54,0q5.39-6,5.38-15.42t-5.38-15.5a18.76,18.76,0,0,0-27.54,0Q430,153.24,430,162.71Z" transform="translate(-76.98 -95.65)"/><path class="cls-2" d="M664.56,160.5v41.6H643V95.65h21.52v37.87q9.75-9.89,21.88-9.89a27.31,27.31,0,0,1,20.44,8.6q8.32,8.61,8.32,23V202.1H693.68V159.92q0-17.79-13-17.79a15.65,15.65,0,0,0-11.26,4.67Q664.56,151.45,664.56,160.5Z" transform="translate(-76.98 -95.65)"/><path class="cls-2" d="M400.28,139.61V124.12h-13V101H365.64v23.1h-9v15.49h9v35.58q0,12.62,7.67,19.94a27.41,27.41,0,0,0,27,6h0V182.76h0c-.59.19-1.19.35-1.78.5a14.53,14.53,0,0,1-3.35.38h-.09a7.11,7.11,0,0,1-5.45-2.51,9.92,9.92,0,0,1-2.3-7V139.61Z" transform="translate(-76.98 -95.65)"/><path class="cls-2" d="M628.45,139.61V124.12h-13V101H593.81v23.1h-9v15.49h9v35.58q0,12.62,7.67,19.94a26.21,26.21,0,0,0,18.79,7.32,26.48,26.48,0,0,0,8.17-1.29h0V182.76h0c-.59.19-1.19.35-1.79.5a14.38,14.38,0,0,1-3.34.38h-.09a7.13,7.13,0,0,1-5.46-2.51,9.92,9.92,0,0,1-2.29-7V139.61Z" transform="translate(-76.98 -95.65)"/></svg>
```

Comment:
This is a different logo than on the previous site (presumably because that one is a combo of graphics and text.  Not a blocker; I'm just noting it in case it should change in a later update.